packages/x-data-grid-pro/src/models/gridRowReorderApi.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * The row reorder API interface.
+ */
+export interface GridRowReorderApi {
+  /**
+   * Sets the row drag active state.
+   * @param {boolean} isActive Whether a row drag operation is currently active.
+   */
+  setRowDragActive: (isActive: boolean) => void;
```

Comment:
Do we need to expose this method? Let's keep it private/undocumented if there's no need for users to use it.